docs/gdd.rst

```diff
@@ -104,7 +104,7 @@ MP Cost
    How much MP is required to use this ability (can be zero)
 
 JP Cost
-   How much JP is required to purchase this ability
+   How much JP is required to purchase this ability (For rough calculation on this value see below)
```

Comment:
We could add a reference so we can link to the calculations.

Reply:
Good idea!